units/host_termination.go

```diff
@@ -391,6 +392,21 @@ func (j *hostTerminationJob) Run(ctx context.Context) {
 			}
 		}
 	}
+
+	provisioningFailedStatuses := []string{evergreen.HostStarting, evergreen.HostProvisioning, evergreen.HostProvisionFailed, evergreen.HostBuilding}
+	if utility.StringSliceContains(provisioningFailedStatuses, prevStatus) && j.host.TaskCount == 0 {
+		event.LogProvisionFailed(j.HostID, fmt.Sprintf("terminating host in status '%s'", prevStatus))
+		grip.Info(message.Fields{
+			"message":     "provisioning failure",
+			"status":      prevStatus,
+			"host_id":     j.HostID,
+			"distro":      j.host.Distro.Id,
+			"uptime_secs": time.Since(j.host.StartTime).Seconds(),
+			"provider":    j.host.Provider,
+			"spawn_host":  j.host.User != evergreen.User,
```

Comment:
`j.host.StartedBy`. `j.host.User` is the login user, like "mci-exec" or whatever.